chia/util/config.py

```diff
@@ -66,10 +72,20 @@ def load_config(
         print("** please run `chia init` to migrate or create new config files **")
         # TODO: fix this hack
         sys.exit(-1)
-    r = yaml.safe_load(open(path, "r"))
-    if sub_config is not None:
-        r = r.get(sub_config)
-    return r
+
+    for i in range(10):
+        try:
+            r = yaml.safe_load(open(path, "r"))
+            if r is None:
+                raise RuntimeError(f"Yaml load on {path} returned None")
+            if sub_config is not None:
+                r = r.get(sub_config)
+            return r
+        except Exception as e:
+            tb = traceback.format_exc()
+            log.error(f"Error loading file: {tb} {e} Retrying {i}")
+            time.sleep(i * 0.1)
```

Comment:
Is the increased back-off intentional, or was it meant to be a constant delay?

Reply:
it's intentional.

Reply:
it's intentional.